{{ cookiecutter.project_name }}/package.json

```diff
@@ -2,45 +2,54 @@
   "name": "{{ cookiecutter.project_name }}",
   "version": "0.0.0",
   "devDependencies": {
-    "babel-eslint": "5.0.0",
-    "babel-plugin-transform-react-jsx": "6.6.4",
-    "eslint": "2.2.0",
-    "eslint-config-defaults": "9.0.0",
-    "eslint-plugin-babel": "3.1.0",
-    "eslint-plugin-react": "4.1.0",
-    "jsdom": "8.1.0",
-    "jsdom-global": "1.6.2",
-    "mocha": "2.4.5",
-    "react-addons-test-utils": "0.14.7",
-    "react-hot-loader": "1.3.0"
   },
   "dependencies": {
     "babel": "6.5.2",
-    "babel-core": "6.6.4",
+    "babel-core": "6.8.0",
+    "babel-eslint": "6.0.4",
     "babel-loader": "6.2.4",
     "babel-plugin-jsx": "1.2.0",
+    "babel-plugin-transform-react-jsx": "6.8.0",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
-    "bourbon-neat": "1.7.3",
+    "babel-preset-stage-1": "6.5.0",
+    "babel-register": "6.8.0",
     "css-loader": "0.23.1",
-    "history": "2.0.1",
-    "node-neat": "1.7.2",
-    "node-sass": "3.4.2",
-    "react": "0.14.7",
-    "react-dom": "0.14.7",
-    "react-ga": "1.2.0",
-    "react-redux": "4.4.0",
-    "react-router": "2.0.0",
-    "redux-actions": "0.9.1",
-    "redux-devtools": "2.1.5",
+    "eslint": "2.9.0",
+    "eslint-config-defaults": "9.0.0",
+    "eslint-plugin-babel": "3.2.0",
+    "eslint-plugin-react": "5.0.1",
+    "fetch-mock": "4.4.0",
+    "history": "2.1.1",
+    "isomorphic-fetch": "2.2.1",
+    "istanbul": "1.0.0-alpha.2",
+    "jsdom": "8.5.0",
+    "jsdom-global": "1.7.0",
+    "lodash": "4.11.2",
+    "mocha": "2.4.5",
+    "node-sass": "3.7.0",
+    "react": "~15.1.0",
+    "react-addons-test-utils": "~15.1.0",
+    "react-dom": "~15.1.0",
+    "react-hot-loader": "1.3.0",
+    "react-ga": "1.4.1",
+    "react-redux": "4.4.5",
+    "react-router": "2.4.0",
+    "redux-asserts": "0.0.7",
     "redux-logger": "2.6.1",
     "redux-thunk": "2.0.1",
-    "sass-loader": "3.1.2",
-    "style-loader": "0.13.0",
-    "webpack": "1.12.14",
+    "sass-loader": "3.2.0",
+    "sinon": "1.17.4",
+    "style-loader": "0.13.1",
+    "url-loader": "0.5.7",
+    "webpack": "1.13.0",
```

Comment:
I think a separate PR would be cleaner, and we should get flow running as part of the micromasters CI first, then do that as part of the work here too.